File: Lecture03/MyApplication/app/src/main/java/com/codingblocks/myapplication/MainActivity.java
package com.codingblocks.myapplication;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final EditText editText = findViewById(R.id.editText);
        final EditText editText2 = findViewById(R.id.editText2);
        final Button button = findViewById(R.id.button);
        final TextView textView = findViewById(R.id.textView);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int a = Integer.valueOf(editText.getText().toString());
                int b = Integer.valueOf(editText2.getText().toString());
                int c = a + b;
                textView.setText(String.valueOf(c));
            }
        });
    }
}

File: Lecture03/MyKtApplication/app/src/main/java/com/codingblocks/myktapplication/MainActivity.kt
package com.codingblocks.myktapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button.setOnClickListener {
            val c = editText.text.toString().toInt() + editText2.text.toString().toInt()
            textView.text = c.toString()

        }
    }
}
